Mage.Sets/src/mage/cards/e/ErietteOfTheCharmedApple.java

```diff
@@ -0,0 +1,127 @@
+package mage.cards.e;
+
+import java.util.Objects;
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.BeginningOfEndStepTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.GainLifeEffect;
+import mage.abilities.effects.common.LoseLifeOpponentsEffect;
+import mage.abilities.effects.common.combat.CantAttackYouAllEffect;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.TargetController;
+import mage.constants.Zone;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.filter.predicate.permanent.ControllerIdPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.targetpointer.FixedTarget;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+
+/**
+ *
+ * @author Xanderhall
+ */
+public final class ErietteOfTheCharmedApple extends CardImpl {
+
+    public ErietteOfTheCharmedApple(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{W}{B}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.HUMAN);
+        this.subtype.add(SubType.WARLOCK);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(4);
+
+        // Each creature that's enchanted by an Aura you control can't attack you or planeswalkers you control.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new ErietteOfTheCharmedAppleEffect()));
+
+        // At the beginning of your end step, each opponent loses X life and you gain X life, where X is the number of Auras you control.
+        Ability ability = new BeginningOfEndStepTriggeredAbility(new LoseLifeOpponentsEffect(ErietteOfTheCharmedAppleValue.instance), TargetController.YOU, false);
+        ability.addEffect(new GainLifeEffect(ErietteOfTheCharmedAppleValue.instance));
```

Comment:
It would be nice to add a `ValueHint` for that X through `ability.addHint(hint)`.

Reply:
Working on that one!